pom.xml

```diff
@@ -120,12 +130,38 @@
       <version>${logback-version}</version>
       <scope>compile</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.kosprov.jargon2</groupId>
+      <artifactId>jargon2-api</artifactId>
+      <version>1.1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.kosprov.jargon2</groupId>
+      <artifactId>jargon2-native-ri-backend</artifactId>
+      <version>1.1.1</version>
+      <scope>runtime</scope>
+    </dependency>
+
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>${junit-version}</version>
+      <groupId>org.jmockit</groupId>
+      <artifactId>jmockit</artifactId>
+      <version>${jmockit-version}</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.jmockit</groupId>
+      <artifactId>jmockit-coverage</artifactId>
+      <version>1.23</version>
+    </dependency>
+
+    <!--    <dependency>-->
```

Comment:
Why do we comment out these lines ?

If we don't need it, just remove it (alone with the property `<junit-version>4.12</junit-version>` below); otherwise, we specify why we keep them in the codebase.

Reply:
Removed.